mtp_cashbook/assets-src/javascripts/main.js

```diff
@@ -22,11 +23,11 @@
     ])
     .init();
 
-  var moduleNames = Object.keys(Mojular.Modules);
-  if(moduleNames.length) {
-    var moduleList = moduleNames.map(function(i) {
-      return '• ' + i;
-    }).join('\n').replace(/^\s+/, '');
-    console.log('The following modules are loaded:\n' + moduleList);
-  }
+  // var moduleNames = Object.keys(Mojular.Modules);
+  // if(moduleNames.length) {
+  //   var moduleList = moduleNames.map(function(i) {
+  //     return '• ' + i;
+  //   }).join('\n').replace(/^\s+/, '');
+  //   console.log('The following modules are loaded:\n' + moduleList);
+  // }
```

Comment:
If we're not using code, we should remove it, not comment it out. That's what git history is for :wink: